src/updateIds.js

```diff
@@ -140,6 +141,19 @@ const parseSuite = suiteTitle => {
   return null;
 };
 
+const fixSuiteContent = (title, replace, content) => {
+  const importLines = content.split('\n').filter(line => line.match(LINE_START_REGEX));
+
+  if (importLines.length) {
+    const testLines = content.split('\n').filter(line => !line.match(LINE_START_REGEX));
+    const updatedTestLines = testLines.map(line => line.replace(title, replace));
+
+    return [...importLines, ...updatedTestLines].join('\n');
```

Comment:
Please place the lines in the very same order they appear in the source code
Do not reorder lines 
It is ok to ignore `const/import/var` lines but they must be placed on the same row position as they were before